### PR TITLE
Restaurant page implementation

### PR DESCRIPTION
**Pull request by ermin-cahtarevic**
- Initialized file structure with webpack
- Separated all of the code into multiple modules 
- Imported all of the modules into index.js 
- Added CSS and file loaders to webpack
- Added three 'tabs' on the homepage
  - When clicked update the current opened 'tab'
- Complied with the styling guides

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1,12 +1 @@
...
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  <div class="content"></div>
-  <script src="main.js"></script>
-</body>
-</html>
\ No newline at end of file
+<!doctype html><html><head><meta charset="utf-8"><title>Burger Shack</title><meta name="viewport" content="width=device-width,initial-scale=1"></head><body><script src="main.js"></script></body></html>
```
Comment (bellom) [inline dist/index.html]: You are missing `<div id="content"></div>`
image
Kindly update the code

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @ermin-cahtarevic  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**1. Kindly check out my comments below;**

**2. Add linter's check and fix all errors:** checkout this [linters-config#troubleshooting](https://github.com/microverseinc/linters-config#troubleshooting) for help. Example below;
image
If you are not having this check on your `Github` page, kindly visit :point_right:[ linter set up](https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml)

**5. You seem to have the config files already:** Let's make sure stickler is ON for this repo on https://stickler-ci.com/
**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: dist/index.html (hunk 1 above)
Comment (ermin-cahtarevic) [reply dist/index.html]: - I've added the missing div tag with the 'content' id.
- And, I've turned on stickler for this repo

**Review by bellom — APPROVED** [review]
## Hi @ermin-cahtarevic  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**